setup.py

```diff
@@ -1,25 +1,28 @@
 '''
 Created on Feb 16, 2017
-
-@author: ask-cafy@cisco.com
 '''
-# sample ./setup.py file
+
 from setuptools import setup
 
 from cafy_pytest.__version__ import __version__
-#__version__ = "0.1.0"  
 
 setup(
     name="cafy_pytest",
     packages=['cafy_pytest'],
     package_data={'cafy_pytest': ['resources/*']},
-    author='Cafy',
```

Comment:
Why is this removed?

Reply:
The "author" field is normally someone's name, rather than a project name. I can add it back if you'd like.